donkey.lua

```diff
@@ -151,6 +151,26 @@ end
 collectgarbage()
 -- End of test loader section
 
+
```

Comment:
also, if classify is given, we dont need to initialize training and testing data loaders